samples/client-sdk/knocking/index.js

```diff
@@ -384,9 +385,22 @@ const handleRejection = (e) => {
   }
 };
 
+const handleGuestJoined = async (e) => {
+  logEvent(e);
+  // Update UI to show they're now in the waiting room
+  hideLoadingText('guest');
+  hideForms('owner');
+  showVideos();
+  showWaitingRoomText();
+  showLeaveButton();
+
+  // Request full access to the call (i.e. knock to enter)
+  await callObject.requestAccess({ name: e?.participants?.local?.user_name });
```

Comment:
How come we need the await here? It doesn't seem like anything contingent on this finishing before we continue or am I missing it?

Reply:
we don't. I like to keep it in demo code to indicate that it's async for the reader but I can remove it.